Makefile

```diff
@@ -281,10 +284,12 @@ test-e2e: docker
 	@docker system prune -f --volumes
 	@echo ">> cleaning e2e test garbage."
 	@rm -rf ./test/e2e/e2e_*
+	@echo ">> Getting gotesplit binary in the tmp dir."
+	scripts/gotesplit.sh -b /tmp/bin
```

Comment:
I was about to ask you why you are not using `bingo` to download this but it seems like there are no publicly available binaries :/

Reply:
Hmm i don't understand. Bingo does not use publicly available binaries, it's meant to pin to a commit in a go module and build the binary. I think bingo would be the right solution here instead of adding this big script just to install gotestsplit.
I'd say let's absolutely use bingo 👍

Reply:
I was so unaware of what Bingo does/could, but yea that is really awesome and the best solution! Implemented this, way better!!!